client/ayon_deadline/abstract_submit_deadline.py

```diff
@@ -442,34 +100,28 @@ def process(self, instance):
 
         assert self._deadline_url, "Requires Deadline Webservice URL"
 
-        file_path = None
-        if self.use_published:
-            if not self.import_reference:
-                file_path = self.from_published_scene()
-            else:
-                self.log.info("use the scene with imported reference for rendering") # noqa
-                file_path = context.data["currentFile"]
-
-        # fallback if nothing was set
-        if not file_path:
-            self.log.warning("Falling back to workfile")
-            file_path = context.data["currentFile"]
-
-        self.scene_path = file_path
-        self.log.info("Using {} for render/export.".format(file_path))
+        job_info = self.get_generic_job_info(instance)
+        self.job_info = self.get_job_info(job_info=job_info)
```

Comment:
I found out the issue but I'm not sure how to describe it. 
The `job_info` variable should include the original values fetched from the deadline profile. and the first call to `self.get_job_info`  should return the overridden values for the export job and save it in `self.job_info`. 

but this somehow modify the `job_info` variable even if calling `get_generic_job_info` no longer retunes the original value.
so we there may be an inner object that is passed by reference causing the functions to mutate the original copy itself. 

so this suggestion fixes the issue on my side: 

```suggestion
        self.job_info = self.get_job_info(job_info=deepcopy(job_info))
```

![image](https://github.com/user-attachments/assets/52d6ca77-78e3-4fd8-bb59-70dbd106af69)